about.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About section for conference website</title>
+    <link rel="stylesheet" href="css/styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+
+        <div class="logo">
+          <a href="./index.html">
+            <img src="./images/logo.png" alt="Company logo" />
+          </a>
+        </div>
+        <ul class="menu">
+          <li class="nav-link"><a href="/index.html">Home</a></li>
+          <li class="nav-link"><a href="">Program</a></li>
+          <li class="nav-link"><a href="">Join</a></li>
+          <li class="nav-link"><a href="">Sponsor</a></li>
+          <li class="nav-link"><a href="">News</a></li>
+          <li class="nav-link"><a href="">Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
- Hey, you're doing great! Please, your header is missing the first top nav. Please, let's add this match the design as much as we can below:

*original design*
![Screenshot from 2022-10-19 09-22-21](https://user-images.githubusercontent.com/59397159/196637819-06992e2e-ef21-4ed8-9aa1-f0c4be16cd07.png)

*your design*
![Screenshot from 2022-10-19 09-21-36](https://user-images.githubusercontent.com/59397159/196637827-73d92cec-97ba-49b2-86f9-dafa180acc04.png)